test/users/npadmana/blas/test_blas3.chpl

```diff
@@ -0,0 +1,519 @@
+use Random;
```

Comment:
Out of curiosity - do these tests reflect the tests in the BLAS repository? If not, that might be an appealing approach for ensuring completeness of testing.